package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "soap",
-  "version": "0.17.0",
+  "version": "0.17.1",
```

Comment:
Please revert this change as we'd love to set the version number ourselves when we release a new version 👍